examples/model_assisted_labeling/video_mal.ipynb

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    project.enable_model_assisted_labeling()
</pre><p>We don't need this anymore :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>